game/tests.py

```
@@ -69,8 +68,23 @@ def test_falls_back_to_python_engine_script(self):
+        super().setUp()
+        self.user = User.objects.create_user(username='testuser', password='password')
+        self.client.login(username='testuser', password='password')
+
+class BoardViewTest(AuthenticatedClientTestCase):
+    """The board page should load and initialise a session for authenticated users."""
+
+    def setUp(self):
+        super().setUp()
+        from django.contrib.auth.models import User
+        self.user = User.objects.create_user(
+            username='testplayer', password='testpass123'
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "password"

(S106)